config/routes.rb

```diff
@@ -26,6 +26,8 @@
 
     resources :activities, only: [], concerns: [:transactionable, :budgetable] do
       resources :steps, controller: "activity_forms"
+      get "extending-organisation/edit", to: "extending_organisations#edit"
```

Comment:
Now that we have a dedicated controller for working with extending organisations. I wonder if we can let Rails take care of the routing in a more conventional way?

```
    resources :activities, only: [], concerns: [:transactionable, :budgetable] do
      resources :steps, controller: "activity_forms"
      resources :extending_organisations, only: [:edit, :update]
    end
```

I think this would change the edit route helper from x_x_edit_path to a familar edit_x_x_path (where edit is the prefix). Rails seems to treat a resource name with multiple words with underscores rather than hyphens too which was interesting.